flink-libraries/flink-python/src/test/java/org/apache/flink/python/api/PythonPlanBinderUnitTest.java

```diff
@@ -56,9 +51,11 @@ private static String findSubproessTestCaseFile() throws Exception {
 				new Path(fs.getWorkingDirectory().toString()
 						+ "/src/test/python/org/apache/flink/python/api"));
 		for (FileStatus f : status) {
-			String file = f.getPath().toString();
-			String[] file_parts = file.split("/");
-			String file_name = file_parts[file_parts.length - 1];
+			Path path = f.getPath();
```

Comment:
On windows, ```Path.getPath()``` returns something like "/C:/hello/world.txt", which mustn't be passed directly to javas ```File``` API.